### DISCUSSION TITLE
Webplayer on mobile too quiet

### DISCUSSION THREAD

**Discussion by Scraggarax** [mcguirepr89/BirdNET-Pi#738] [General]
I've noticed when I play any recoding or live stream from birdnet pi on my android phone it comes through the phone speaker as if I'm on a call, rather than the loudspeaker like it's a video or regular media. This means that I, or anyone I'm showing birdnet to, typically has to put it to their ear to hear it. Is this something that can be fixed through coding? The default player output? Or is this an android issue?

**Comment by ehpersonal38** [comment]
I'm afraid I can't replicate this. The audio stream uses a native `<audio>` html element, there's nothing special there. I would try it on a different browser and see if that helps
